{
	"candidate": "CVE-2019-20142",
	"description": "An issue was discovered in GitLab Community Edition (CE) and Enterprise Edition (EE) 12.3 through 12.6.1. It allows Denial of Service.",
	"notes": {
		"msalvatore": "Affects GitLab CE/EE 12.3 and later."
	},
	"priority": "medium",
	"cvss": {
		"nvd": " CVSS:3.1/AV:N/AC:L/PR:L/UI:N/S:U/C:N/I:N/A:L [4.3 MEDIUM]"
	},
	"references": [
		"https://about.gitlab.com/blog/2020/01/02/security-release-gitlab-12-6-2-released/",
		"https://www.cve.org/CVERecord?id=CVE-2019-20142"
	],
	"public_date": "2020-01-13 21:15:00 UTC",
	"packages": {
		"bionic": {
			"gitlab": {
				"name": "gitlab",
				"status": "DNE"
			}
		},
		"devel": {
			"gitlab": {
				"name": "gitlab",
				"status": "DNE"
			}
		},
		"disco": {
			"gitlab": {
				"name": "gitlab",
				"status": "DNE"
			}
		},
		"eoan": {
			"gitlab": {
				"name": "gitlab",
				"status": "DNE"
			}
		},
		"esm-apps/xenial": {
			"gitlab": {
				"name": "gitlab",
				"status": "not-affected",
				"note": "code not present"
			}
		},
		"esm-infra-legacy/trusty": {
			"gitlab": {
				"name": "gitlab",
				"status": "DNE"
			}
		},
		"precise/esm": {
			"gitlab": {
				"name": "gitlab",
				"status": "DNE"
			}
		},
		"trusty": {
			"gitlab": {
				"name": "gitlab",
				"status": "DNE"
			}
		},
		"trusty/esm": {
			"gitlab": {
				"name": "gitlab",
				"status": "DNE"
			}
		},
		"upstream": {
			"gitlab": {
				"name": "gitlab",
				"status": "not-affected",
				"note": "debian: Only affects Gitlab CE 12.3 and later"
			}
		},
		"xenial": {
			"gitlab": {
				"name": "gitlab",
				"status": "not-affected",
				"note": "code not present"
			}
		}
	}
}
